### PR TITLE
Fixed display of prediction in automl-image-classification-multilabel…

### PR DESCRIPTION
**Pull request by jeff-shepherd**
…-task-fridge-items-mlflow

# PR into Azure/azureml-examples

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

- Fixed display of prediction in automl-image-classification-multilabel-task-fridge-items-mlflow
- The value returned from the service is a list of predictions.
- In this case only one image was passed and so only one prediction is used

fixes #

### CODE REVIEW

**Review by rtanase — APPROVED** [review]

File: sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items-mlflow.ipynb [hunk 1]
```diff
@@ -1160,7 +1161,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.10"
+   "version": "3.8.12"
```
Comment (needuv) [inline sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items-mlflow.ipynb:1164]: is this change intentional?

Comment (jeff-shepherd) [reply sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items-mlflow.ipynb:1164]: Yes.  It is not very significant, but we do test on Python 3.8
